.github/workflows/format.yml

```diff
@@ -44,4 +44,4 @@ jobs:
         -DCMAKE_CXX_STANDARD=17
 
     - name: Build
-      run: cmake --build build -j 2
+      run: cmake --build build -j 2 -- --keep-going
```

Comment:
Thanks. It was indeed super annoying to not get all the clang-tidy errors.

Reply:
@Skylion007 Do you prefer to adopt this in your PR #3051, or do you think it's better if I open a mini PR to add this to master?

Reply:
Adopting